LICENSE

```diff
@@ -5,6 +5,7 @@ Copyright (c) 2021 Andreas Stuerz
 Copyright (c) 2021 Axelrtgs
 Copyright (c) 2023 Bernhard Frenking <bernhard@frenking.eu>
 Copyright (c) 2023 Cannon Matthews <cannonmatthews@google.com>
+Copyright (c) 2023-2024 Cedrik Pischem <cedrik@pischem.com>
```

Comment:
this is automatically updated through "make sync" .. which would also fix the README so you added this by hand unnecessarily ;)

Reply:
Hehe okay, didn't know that. 1fc8ed2